json_schemas/test_story.schema.yaml

```diff
@@ -163,3 +165,11 @@ definitions:
       - type: string
       - type: number
       - type: boolean
+
+  Warnings:
+    type: object
+    properties:
+      multiple-paths-detected:
+        type: boolean
+        default: true
```

Comment:
It's unclear what turning this off and on means. Please add a description for this property.